rtos/rtx/TARGET_CORTEX_M/RTX_CM_lib.h

```diff
@@ -648,6 +649,7 @@ __asm void __rt_entry (void) {
   LDR     R4,=armcc_heap_top
   STR     R0,[R3]
   STR     R2,[R4]
+  BL      _platform_post_stackheap_init
```

Comment:
Does this need to be called before the RTOS has been initialized? If not you might want to move this to pre_main.